nighthawk/include/nighthawk/client/output_formatter.h

```diff
@@ -14,8 +14,13 @@ namespace Client {
 class OutputFormatter {
 public:
   virtual ~OutputFormatter() = default;
+  virtual void addResult(const std::string name, const std::vector<StatisticPtr>& statistics,
```

Comment:
`absl::string_view name`? (or make it a ref?)

Reply:
done